packages/template-retail-react-app/app/pages/checkout/partials/contact-info.jsx

```diff
@@ -62,9 +67,20 @@ const ContactInfo = ({isSocialEnabled = false, isPasswordlessEnabled = false, id
     const [showPasswordField, setShowPasswordField] = useState(false)
     const [signOutConfirmDialogIsOpen, setSignOutConfirmDialogIsOpen] = useState(false)
 
+    const [authModalView, setAuthModalView] = useState(PASSWORD_VIEW)
+    const authModal = useAuthModal(authModalView)
+    const [isPasswordlessLoginClicked, setIsPasswordlessLoginClicked] = useState(false)
+
     const submitForm = async (data) => {
         setError(null)
         try {
+            if (isPasswordlessLoginClicked) {
+                await authorizePasswordlessLogin.mutateAsync({userid: data.email})
+                setAuthModalView(EMAIL_VIEW)
```

Comment:
For some reason the screen reader does not announce the Check Email text like when logging in via the auth modal.

After some digging, it seems like it's because the focus is set immediately to the close button which prevents the screen reader from reading out the text within the `role="alert"`. I also noticed the focus is not getting set to the first form field as wanted by [this](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/50c0f74e7fece8aed843e9a840765cd01c1d0c0c/packages/template-retail-react-app/app/hooks/use-auth-modal.js#L175-L183). 

Thinking we should create a new ticket to capture this. Thoughts?